### PR TITLE
docs/cdn-links.md

### PR DESCRIPTION
**Pull request by ph1p**

### CODE REVIEW

File: content/docs/cdn-links.md [hunk 1]
```diff
@@ -6,32 +6,33 @@ prev: create-a-new-react-app.html
 next: hello-world.html
 ---
 
-Both React and ReactDOM are available over a CDN.
+React und ReactDOM sind beide über [CDN](/docs/glossary.html#cdn)s verfügbar.
```
Comment (miku86) [inline content/docs/cdn-links.md]: Hey ph1p,
you added a link to the CDN glossary site, that isn't in the original english site.
I actually don't know if this is desired.
@tesseralis @gaearon : What do you think about it?

Comment (ph1p) [reply content/docs/cdn-links.md]: I've removed it

Comment (tesseralis) [reply content/docs/cdn-links.md]: Hmm... It's up to you. If you don't think "CDN" is obvious enough in german you can do something like "available over a Content Domain Network (CDN)".

Comment (ph1p) [reply content/docs/cdn-links.md]: I'm gonna leave it like this and if someone doesn't know it, he can find it at the glossary.
What amazes me is that you have written Content Domain Network, but everywhere it is called Content Delivery Network.

Comment (tesseralis) [reply content/docs/cdn-links.md]: I don't actually remember what CDN stands for. I mean, who uses CDN these days right? 🤷‍♀️

File: content/docs/cdn-links.md [hunk 2]
```diff
@@ -6,32 +6,33 @@ prev: create-a-new-react-app.html
...
-Both React and ReactDOM are available over a CDN.
+React und ReactDOM sind beide über CDNs verfügbar.
 
 ```html
 <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
 <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
 ```
 
-The versions above are only meant for development, and are not suitable for production. Minified and optimized production versions of React are available at:
+Diese beiden Versionen sind nur zu Entwicklung angedacht und nicht für den Live-Einsatz in Produktionsystemen.
```
Comment (ChrisB9) [inline content/docs/cdn-links.md]:
```suggestion
Diese beiden Versionen sind nur zu Entwicklung angedacht und nicht für den Live-Einsatz in Produktivsystemen.
```

**Pull request merged**